department_app/rest/address_api.py

```diff
@@ -0,0 +1,153 @@
+"""Rest addresses Api"""
+from flask import Blueprint, request, jsonify, make_response
+from flask_restful import Resource, Api, abort
+from pydantic import ValidationError
+from department_app.models.app_models import Address
+from department_app.schemas.address_schema import AddressModel
+from department_app.models.app_models import db
+
+address_api = Blueprint('address_api', __name__)
+
+api = Api(address_api)
+
+
+class AddressInfo(Resource):
+    """Address API class"""
+    @staticmethod
+    def get(address_id):
+        """
+        This is the Address API
+        Call this API passing a address_id and get back address information
+        ---
+        tags:
+          - Address API
+        parameters:
+          - name: "address_id"
+            in: "path"
+            description: "ID of address to return"
+            required: true
+            type: "integer"
+            format: "int64"
+        responses:
+          404:
+            description: Could not find address
+          200:
+            description: Address information returned
+        """
+        if not str(address_id).isdigit():
```

Comment:
As I understand you have implemented validation using Pydantic, right? Can we now use these classes (pydantic) to validate requests?